src/main/resources/db/migration/V2022_09_30__YSMOD-636-behandling-endret-tidspunkt.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE behandling ADD COLUMN endret_tidspunkt TIMESTAMP;
+
+-- Vi har ikke tidspunktet for siste endring, så bruker opprettet tidspunkt for å sette en verdi på behandlinger som er endret
+UPDATE behandling SET endret_tidspunkt = opprettet_tidspunkt where status <> 'IKKE_PAABEGYNT';
```

Comment:
En liten tanke her; Siden endret_tidspunkt er NULLABLE, og vi ikke vet når endret_tidspunkt er, så kan det vel kanskje være unødvendig å sette denne verdien? Vi har uansett ikke kommet til et punkt i prod hvor behandlinger har endret status, og denne migreringen vil kun gjelde for dev

Reply:
ok. kan ta bort denne updaten